secant/Features/SendFlow/SendFlowStore.swift

```diff
@@ -287,7 +286,7 @@ extension SendFlowState {
         .init(
             route: nil,
             transaction: .init(
-                amount: 0,
+                amount: Zatoshi(),
```

Comment:
After reading this it made me think if a helper static variable would make this more readable like CGRect or CGSize have their "make zero" helpers.

`Zatoshi.zero`


what do you think?

Reply:
Implemented